docs/en/index.md

```diff
@@ -141,3 +141,44 @@ $paginator->setItemsPerPage(500);
 
 The first shown record will be maintained across page size changes, and the number of pages and current page will be
 recalculated on each request, based on the current first shown record and page size.
+
+
+Actions Menu
+------------
+
+The `GridFieldActionsMenu` serves as an encapsulation type component, which for each item/row will render a pop-out/dropdown menu to the user - represented on in the UI as the 'meatball' type widget (as opposed to the ever popular 'hamburger' collapsing menu widget). The actions menu will render one of three classes of information:
+
+ - **Links to different edit screens**
+   If the item in the current row has an edit form (`getCMSFields`) which contains a root level `TabSet`, this division of items houses an entry for each of it's contained `Tab`s in the nature of e.g. `GridFieldEditButton`, etc. A parameter of `false` can be passed in to skip the first tab, in the case that clicking the row should invoke this action (such as the aforementioned edit button does).
+ - **Versioned actions**
+   Iff the item is versioned (via the `Versioned` extension), then applicable actions will be rendered in this division. These will be _Publish_, _Unpublish_, and _Archive_ (aka delete). It is worth noting that Publish and Unpublish are _not_ mutually exclusive, as a record can have changes ahead of it's published version (i.e. "Modified" as opposed to "Draft").
```

Comment:
```diff
- Iff
+ If
```

Reply:
Nah, it's a thing :)
> In logic and related fields such as mathematics and philosophy, if and only if (shortened iff) is a biconditional logical connective between statements.

https://en.wikipedia.org/wiki/If_and_only_if

Reply:
Do you think it's appropriate to use in the context of this documentation and its audience?

Reply:
In developer documentation I'd have thought so, yes.